Add Finout Tile for the Finout Datadog Application (#1240)
DataDog/integrations-extras: finout/README.md

Finout’s Datadog app allows you to add cost widgets into your Datadog dashboards and combine your AWS, Snowflake, Kubernetes, and Unit costs alongside your usage.

## Setup

### Installation

If you don't have a Finout account yet, [sign up](https://finout.io/start-now) for a free trial. After you accept the account invitation, you are set to start working with the Finout Application in Datadog, no code needed. 

For a Kubernetes-level cost breakdown or a unit cost based on a Datadog metric, contact Finout Support in the Finout application or email [support@finout.io](mailto:support@finout.io).

### Configuration

1. Install the app from the Datadog tile.
2. Add the Finout widget to your dashboard.
3. Authenticate your Finout account.
4. Configure the following widget parameters:
    - Usage type (Cost, Total Coverage, RI, SP, Spot, or On-Demand)
    - Chart type (Bar or Line)
    - Context (Select the Finout business context you want to visualize).
    - View (Select which Finout saved view you want to visualize).
5. Continue adding more Finout widgets!

> ## Troubleshooting

Reviewer: ```suggestion
## Support
```

# Finout: The Modern cost Observability platform for FinOps

## Overview

[Finout](https://finout.io) combines Cloud Providers, Data Warehouses, and CDNs  
into one mega bill, enabling an unparalleled business context view of your cloud spend with no heavy lifting in minutes.

> Finout’s cloud cost management tool uses your container metrics in order to break down Kubernetes cost without the need to install any agent, and also connect to any metric within Datadog to create a unit cost from it, saving countless hourls and improving cost observability company wide 

Reviewer 2: ```suggestion
Finout’s cloud cost management tool uses your container metrics in order to break down Kubernetes costs without the need to install an agent. You can connect to any metric within Datadog and use it to create a unit cost, saving countless hours and improving your cost observability company-wide.
```

> Finout’s cloud cost management tool uses your container metrics in order to break down Kubernetes costs without the need to install an agent. You can connect to any metric within Datadog and use it to create a unit cost, saving countless hours and improving your cost observability company-wide.

Reviewer 3: ```suggestion
Finout’s cloud cost management tool uses your container metrics to break down Kubernetes costs without installing an agent. You can connect to any metric within Datadog and create a unit cost to improve cost observability.
```

> Finout’s Datadog application allows you to add cost widgets into your Datadog dashboards, combining your AWS, Snowflake, Kubernetes, and Unit costs alongside your usage in one place.

Reviewer 3: ```suggestion
Finout’s Datadog integration allows you to add cost widgets into your Datadog dashboards and combine your AWS, Snowflake, Kubernetes, and Unit costs alongside your usage.
```

## Setup

### Installation

> If you don't already have a Finout account, [sign up](https://finout.io/start-now) for your free trial.
> Once you get the invitation to your account, you are all set to start working with the Finout Application in Datadog within minutes, no code needed. 

Reviewer 3: ```suggestion
If you don't have a Finout account yet, [sign up](https://finout.io/start-now) for a free trial. After you accept the account invitation, you are set to start working with the Finout Application in Datadog, no code needed. 
```

> Once you get the invitation to your account, you are all set to start working with the Finout Application in Datadog. No code needed - two minutes of intergration

Reviewer 2: ```suggestion
Once you get the invitation to your account, you are all set to start working with the Finout Application in Datadog within minutes, no code needed. 
```

> To get Kubernetes level cost breakdown or unit cost that is based on Datadog metric, please reach out to Finout support directly in the Finout application or here: (mailto:support@finout.io) .

Reviewer 2: ```suggestion
To get a Kubernetes-level cost breakdown, or a unit cost that is based on a Datadog metric, please reach out to Finout support directly in the Finout application or by emailing us at [support@finout.io](mailto:support@finout.io).
```

> To get a Kubernetes-level cost breakdown, or a unit cost that is based on a Datadog metric, please reach out to Finout support directly in the Finout application or by emailing us at [support@finout.io](mailto:support@finout.io).

Reviewer 3: ```suggestion
For a Kubernetes-level cost breakdown or a unit cost based on a Datadog metric, contact Finout Support in the Finout application or email [support@finout.io](mailto:support@finout.io).
```

### Configuration

> 1. Install the integration

Reviewer 2: ```suggestion
1. Install the app from this tile
```

2. Add Finout widget to your dashboard

> 3. Authenticate with your Finout account - 

Reviewer 2: ```suggestion
3. Authenticate with your Finout account
```

> 1. Install the app from this tile
> 2. Add the Finout widget to your dashboard
> 3. Authenticate with your Finout account
> 4. Configure the widget parameters
> 	1. Usage type (Cost, Total Coverage, RI, SP, Spot, or On-Demand)
> 	2. Chart type (Bar or Line)
> 	3. Context (Choose the Finout business context you want to visualize)
> 	4. View (Which Finout saved view do you want to visualize)
> 5. Add as many Finout widgets as you like!

Reviewer 3: ```suggestion
1. Install the app from the Datadog tile.
2. Add the Finout widget to your dashboard.
3. Authenticate your Finout account.
4. Configure the following widget parameters:
    - Usage type (Cost, Total Coverage, RI, SP, Spot, or On-Demand)
    - Chart type (Bar or Line)
    - Context (Select the Finout business context you want to visualize).
    - View (Select which Finout saved view you want to visualize).
5. Continue adding more Finout widgets!
```

## Troubleshooting

> Please note, that when looking at AWS cost you will probably need to increase the timeframe of your query for at least a week, as the AWS cost and usage report is always two days behind.

Reviewer 2: ```suggestion
Please note that when looking at AWS cost you may need to expand the timeframe of your query to at least a week, as AWS cost and usage reporting is always two days behind.
```

> Please note that when looking at AWS cost you may need to expand the timeframe of your query to at least a week, as AWS cost and usage reporting is always two days behind.

Reviewer 3: ```suggestion
**Note:** When looking at AWS cost, you may need to expand the timeframe of your query to at least one week because AWS cost and usage reporting are always two days behind.
```

Any questions? We are here to help!

> Just drop us an email in [support@finout.io](mailto:support@finout.io) or use the in-app chat.

Reviewer 2: ```suggestion
Just drop us an email at [support@finout.io](mailto:support@finout.io) or use the in-app chat.
```

> Any questions? We are here to help!
> Just drop us an email at [support@finout.io](mailto:support@finout.io) or use the in-app chat.

Reviewer 3: ```suggestion
Any questions? We are here to help! Email [support@finout.io](mailto:support@finout.io) or use the in-app chat.
```

# Agent Check: Finout

## Overview

This check monitors [Finout][1].

## Setup

### Installation

To install the Finout check on your host:

1. Install the [developer toolkit]
(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
 on any machine.

2. Run `ddev release build finout` to build the package.

3. [Download the Datadog Agent][2].

4. Upload the build artifact to any host with an Agent and
 run `datadog-agent integration install -w
 path/to/finout/dist/<ARTIFACT_NAME>.whl`.

### Configuration

1. <List of steps to setup this Integration>

### Validation

<Steps to validate integration is functioning as expected>

## Data Collected

### Metrics

Finout does not include any metrics.

### Service Checks

Finout does not include any service checks.

### Events

Finout does not include any events.

## Troubleshooting

Need help? Contact [Datadog support][3].

> [1]: **LINK_TO_INTEGRATION_SITE**

Reviewer 2: Calling out that this is a missing link :)
